### PR TITLE
[DISKPART] Update Polish (pl-PL) translation v2

### PR DESCRIPTION
**Pull request by pithwz**
diskpart update help in lang pl-pl v2

### CODE REVIEW

File: base/system/diskpart/lang/pl-PL.rc [hunk 1]
```diff
@@ -154,54 +154,54 @@ END
...
-    IDS_HELP_CREATE_PARTITION_EFI      "Create an EFI system partition.\n"
-    IDS_HELP_CREATE_PARTITION_EXTENDED "Create an extended partition.\n"
-    IDS_HELP_CREATE_PARTITION_LOGICAL  "Create a logical drive.\n"
-    IDS_HELP_CREATE_PARTITION_MSR      "Create an MSR partition.\n"
-    IDS_HELP_CREATE_PARTITION_PRIMARY  "Create a primary partition.\n"
-    IDS_HELP_CREATE_VOLUME             "Create a volume.\n"
-    IDS_HELP_CREATE_VDISK              "Create a virtual disk file.\n"
+    IDS_HELP_CREATE_PARTITION          "Utwórz partycję.\n"
+    IDS_HELP_CREATE_PARTITION_EFI      "Tworzy partycję systemową EFI.\n"
+    IDS_HELP_CREATE_PARTITION_EXTENDED "Utwórz partycję rozszerzoną.\n"
```
Comment (Saibamen) [inline base/system/diskpart/lang/pl-PL.rc:171]: No consistency for `Create` translation

File: base/system/diskpart/lang/pl-PL.rc [hunk 2]
```diff
@@ -154,54 +154,54 @@ END
...
+    IDS_HELP_DELETE_PARTITION          "Usuń wybraną partycję.\n"
+    IDS_HELP_DELETE_VOLUME             "Usuń wybrany wolumin.\n"
 
     IDS_HELP_DETACH                    "Odłącza wirtualny dysk.\n"
 
     IDS_HELP_DETAIL                    "Podaj szczegółowe informacje o obiekcie.\n"
-    IDS_HELP_DETAIL_DISK               "Print disk details.\n"
-    IDS_HELP_DETAIL_PARTITION          "Print partition details.\n"
-    IDS_HELP_DETAIL_VOLUME             "Print volume details.\n"
+    IDS_HELP_DETAIL_DISK               "Wyświetla właściwości wybranego dysku.\n"
```
Comment (Saibamen) [inline base/system/diskpart/lang/pl-PL.rc:186]: `Wyświetl`?

**Review by Saibamen — CHANGES_REQUESTED** [review]

**Comment by binarymaster** [comment]
I've fixed merge conflict and some inconsistencies, cc @reactos/lang-polish for final review.

File: base/system/diskpart/lang/pl-PL.rc [hunk 3]
```diff
@@ -214,11 +214,11 @@ BEGIN
...
-    IDS_HELP_SELECT                    "Przenieś fokus na jeden z obiektów.\n"
-    IDS_HELP_SELECT_DISK               "Moves the focus to the disk.\n"
-    IDS_HELP_SELECT_PARTITION          "Moves the focus to the partition.\n"
-    IDS_HELP_SELECT_VOLUME             "Moves the focus to the volume.\n"
-    IDS_HELP_SELECT_VDISK              "Moves the focus to the virtual disk.\n"
+    IDS_HELP_SELECT                    "Przenieś fokus na obiekt.\n"
+    IDS_HELP_SELECT_DISK               "Przenieś fokus na dysk.\n"
+    IDS_HELP_SELECT_PARTITION          "Przenieś fokus na partycję.\n"
+    IDS_HELP_SELECT_VOLUME             "Przenieś fokus na wolumin.\n"
+    IDS_HELP_SELECT_VDISK              "Przenieś fokus na dysk wirtualny.\n"
```
Comment (Saibamen) [inline base/system/diskpart/lang/pl-PL.rc:221]: I'm not sure about it. Can you send screenshot from PL WinXP?

Comment (archeYR) [reply base/system/diskpart/lang/pl-PL.rc:221]: Seems fine to me. Search "Move focus" on MS Language Portal.

**Review by binarymaster — CHANGES_REQUESTED** [review]
It seems you have hidden email for GitHub commits:
- https://git.reactos.org/?p=reactos.git;a=commit;h=b4dedecff34ec332d2b9906607776e1164e9bdfd
```
Piotr Hetnarowicz <40692062+pithwz@users.noreply.github.com>
```

Please uncheck `Keep my email addresses private` in [GitHub email settings](https://github.com/settings/emails) so I can merge without this problem.

Let me know when it's done.

**Review by Saibamen — APPROVED** [review]

**Review by binarymaster — APPROVED** [review]

**Pull request merged**